activerecord/test/cases/connection_adapters/legacy_connection_handlers_multi_db_test.rb

```diff
@@ -392,7 +396,9 @@ def test_connection_handlers_are_per_thread_and_not_per_fiber
       end
 
       def test_connection_handlers_swapping_connections_in_fiber
-        ActiveRecord::Base.connection_handlers = { writing: ActiveRecord::Base.default_connection_handler, reading: ActiveRecord::ConnectionAdapters::ConnectionHandler.new }
+        assert_deprecated do
+          ActiveRecord::Base.connection_handlers = { writing: ActiveRecord::Base.default_connection_handler, reading: ActiveRecord::ConnectionAdapters::ConnectionHandler.new }
```

Comment:
```suggestion
          ActiveRecord::Base.connection_handlers = {
            writing: ActiveRecord::Base.default_connection_handler,
            reading: ActiveRecord::ConnectionAdapters::ConnectionHandler.new
          }
```

Reply:
Thanks for the suggestions. I'm not going to merge in the suggested changes because it's deprecated code, and they're just tests.